### ISSUE TITLE
[ruby] False positive with indentation validation?

### ISSUE THREAD

**Issue by abeger** [exercism/rikki#37]
This code got flagged for inconsistent indentation and I cannot figure out why: 

http://exercism.io/submissions/7fe9078c64fd4cb5a1b2072270ab4a74

Is there something else wrong with the code and the message is misleading?

**Comment by Insti** [comment]
I can't see any problems with the indentation.

My guess is that the `begin` on line 27 that's causing it.

But there are other reasons why you don't want to be doing that.

**Comment by kytrinyx** [comment]
Agreed—we shouldn't be flagging that `begin` as inconsistent indentation.

Any chance you'd have time to write a failing test for this for https://github.com/JacobNinja/exercism-analysis? Having a minimal code example that proves exactly what the codeminer is unhappy about would be super helpful towards getting a fix in.

**Comment by abeger** [comment]
Sure I'll take a swing at it

**Comment by abeger** [comment]
As something of an aside, has there been any talk of using a custom Rubocop configuration for the Ruby analyzer? `exercism-analysis` hasn't been updated in 3 years and is getting a little out of date.

**Comment by kytrinyx** [comment]
@abeger That's a great point. I've used Rubocop to write some linters and do some static analysis, and it's a good tool.

We won't be using rikki- directly in the [v2](https://github.com/exercism/v2-feedback/blob/master/README.md), rather we'll be writing some new tools for mentors. I think it would be worth considering Rubocop for the new tools.

**Comment by abeger** [comment]
How far away is v2? Would it be worth my time to experiment with making Rikki RuboCop-based?

**Comment by kytrinyx** [comment]
@abeger we don't actually know—we hope not too far, but my guess would be more than weeks. My gut sense is "probably worth holding off", though I will start thinking about what v2's rikki would look like so we could potentially start working on that.
